src/components/Routes/ClientRoutes.js

```diff
@@ -1,15 +1,29 @@
-import React from "react";
-import { Routes, Route } from "react-router-dom";
-import Wallet from "../../pages/Wallet/Wallet";
-import TransferStatus from "../../pages/TransferStatus/TransferStatus";
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import SendTokens from '../../pages/SendTokens/SendTokens';
+import TransferStatus from '../../pages/TransferStatus/TransferStatus';
+import Wallet from '../../pages/Wallet/Wallet';
 
 const ClientRoutes = () => {
-  return (
-    <Routes>
-      <Route path="/" exact element={<Wallet />} />
-      <Route path="/transfer-status" exact element={<TransferStatus />} />
-    </Routes>
-  );
+	return (
+		<Routes>
+			<Route
+				path="/"

```

Comment:
Am I right that the side menu is collapsed by default only in mobile view? In the desktop version, we have plantly of space so I think it should display in full size.